docs/090_roadmap/README.md

```diff
@@ -1,5 +1,53 @@
 # Roadmap for Pepr
 
+_2025 Roadmap_  
+
+## Phase 1: Code Quality - Experimentation
+
+- **Q1**:
+  - **Turn on eslint enforcement and configure settings and see no warnings**:
+    - Eliminate circular dependencies, complexity, return statements, etc.
+  - **Metric and Performance Baselining**:
+    - Establish a baseline for performance and resource utilization metrics. Use this data to make informed decisions about the direction of the project in terms of Deno2
+  - **Nightly Release**:
+    - Establish a nightly release process. This will help us to catch bugs early and ensure that the project is always in a releasable state.
+
+## Phase 2: Durable Storage for Metrics and Performance Tests / Transactional Pepr Store
+
+- **Q2**:
+  - **Professional Dashboard displaying metrics and performance tests originating from CI**:
+  - **Determine if a Transactional PeprStore makes sense**:
+    - Sus out details involved with having a transactional Pepr Store. What are the implications of this? What are the benefits? What are the drawbacks? What are the use-cases? What are the technologies that can be used to implement this?
+  - **Define Governance with Pepr**
+    - Add support for compliance/governance validation within Pepr. See [pepr/#1823](https://github.com/defenseunicorns/pepr/issues/1823)
+
+## Phase 3: TBD
+
+- **Q3**:
+  - **Support Governance with Pepr**
+    - Improve initial support added in Q2.
+  - **Transactional PeprStore Implementation**:
+    - Begin integrating transactional functionality into PeprStore. The implementation will emphasize robust testing and clear documentation to support fast and reliable data operations in a transactional manner.
+
+## Phase 4: TBD
+
+- **Q4**:
+  - **Prepare for Project Donation**
+    - Consider Pepr project donation after resolving the open topics of Governance and performance at scale
+  - **Experimentation with Deno2**:
+    - Experiment with Deno2 through Dash Days and see if it can be used in the project. Look into the performance improvements and new features that Deno2 brings to the table.
+
+## Deferred or Unprioritized Work
+
+This section contains work we've considered, but have not slotted in to the roadmap or have had to deprioritize.
+
+- **Pepr v1.0.0**
+  - Determine if Pepr is stable enough to be at `v1.0.0` and release it!
+- **OTEL Preparation**:
+  - Come up with a plan to implement Open Telemetry. Specifically distributed tracing, metrics, logs and events. Use this data to make debugging easier from a UDS Core prespective. There will be documentation work on how to use an OTEL collector with a Pepr Module.
+- **Deno2 Implementation**:
```

Comment:
based on the fact that we have had good and stable performance this lift would likely not make sense in Q4

Reply:
Moved to deprioritized work